posthog/api/test/test_insight.py

```diff
@@ -2003,165 +1985,6 @@ def test_admin_user_can_add_an_insight_to_a_restricted_dashboard(self) -> None:
         )
         assert response.status_code == status.HTTP_200_OK
 
-    def test_saving_an_insight_with_new_filters_updates_the_dashboard_tile(
```

Comment:
I guess these are covered in the `InsightCachingState` tests?

Reply:
Yes!